### ISSUE TITLE
[Feature request] Release apk files & publish on izzy F-droid repo

### ISSUE THREAD

**Issue by shuvashish76** [RoepStoep/lidrobile#2]
Please publish apk files just like lichess releases: https://github.com/veloce/lichobile/releases
Lichess app already available on F-Droid: https://f-droid.org/packages/org.lichess.mobileapp (Izzy repo needs to be enabled)
Izzy repo link: https://apt.izzysoft.de/fdroid/index/apk/org.lichess.mobileapp

It would be nice to have lidraughts app on F-droid :)

Requirements: https://gitlab.com/IzzyOnDroid/repo

![image](https://user-images.githubusercontent.com/48226312/129659178-49ea37dc-fd5e-45fa-92e8-a48a7cf44737.png)

**Comment by shuvashish76** [comment]
IzzyOnDroid repo issue: https://gitlab.com/IzzyOnDroid/repo/-/issues/181

ping @RoepStoep

**Comment by RoepStoep** [comment]
After a long break I've returned to development for lidrobile. A new version has just been released:
https://github.com/RoepStoep/lidrobile/releases/tag/v2.1.0

I'm not at all familiar with F-droid, but I'd be happy to help make the lidraughts app available there. I've now included the same script as lichess used to build an apk without Firebase, which is probably required. See e.g. the freeApk job in https://github.com/RoepStoep/lidrobile/blob/master/.github/workflows/android.yml

Regarding app size, the Scan 3.1 engine requires a 8Mb data-file for each draughts variant. As there are currently 4 variants the apk size will always be larger than 30Mb.
